api/httpparameters/get_test.go

```diff
@@ -29,3 +30,17 @@ func TestQueryParamWithDefault(t *testing.T) {
 	})
 
 }
+
+func TestGetFilterQueryParamWithOutPrefix(t *testing.T) {
+	queryPrefix := "bla_"
+	expectedValue := "stays"
+	v := url.Values{}
+	v.Set(fmt.Sprintf("bla_%s", expectedValue), "value")
```

Comment:
For better testing, can you add a query parameter without `bla` ? (to check that the field not returned)

Reply:
Changed.